kvmagent/kvmagent/plugins/ft_vm_fencer.py

```
@@ -85,7 +85,7 @@ def do_host_disk_raid_state_fencer():
                     continue
                 elif "DiskGroup" in info:
                     kvs = info.replace("Drive's position: ", "").split(",")
-                    disk_group = filter(lambda x: "DiskGroup" in x, kvs)[0]
+                    disk_group = [x for x in kvs if "DiskGroup" in x][0]
```

Ruff (0.15.10)

[warning] Prefer `next(x for x in kvs if "DiskGroup" in x)` over single element slice

Replace with `next(x for x in kvs if "DiskGroup" in x)`

(RUF015)